tests/warp/aggregator/aggregator.go

```diff
@@ -86,7 +86,10 @@ func (a *Aggregator) AggregateSignatures(ctx context.Context, unsignedMessage *a
 					"err", err,
 					"msgID", unsignedMessage.ID(),
 				)
-				signatureFetchResultChan <- nil
+				select {
+				case <-signatureFetchCtx.Done():
+				case signatureFetchResultChan <- nil:
+				}
```

Comment:
Why are these select cases modified? Now that the length of the channel is sufficient, these are guaranteed never to block right?